packages/paste-core/components/select/__tests__/option.test.tsx

```diff
@@ -1,14 +1,19 @@
 import * as React from 'react';
 import {render} from 'react-dom';
-import {render as testRender} from '@testing-library/react';
+import {matchers} from 'jest-emotion';
+import {render as testRender, screen} from '@testing-library/react';
+import {CustomizationProvider} from '@twilio-paste/customization';
 // @ts-ignore typescript doesn't like js imports
 import axe from '../../../../../.jest/axe-helper';
 import {Option} from '../src';
 import {createAttributeMap} from '../test-utils';
 
+expect.extend(matchers);
+
 interface MockOptionProps extends React.OptionHTMLAttributes<HTMLOptionElement> {
   suffix?: string;
   value?: string;
+  element?: string;
```

Comment:
Is it possible to use non-mocked props for this? or do we want them mocked?

Reply:
I should probably rename this component function here, as it's not actually mocking anything, rather just creating a render of the component that we want to test configured for the test. 

That said, the interface for that component's props is called `MockOptionProps` because it's using a naming convention where the props interface is named by simply adding `Props` as a suffix to the component name.

Does that answer the question? Would renaming the props interface here make this more clear?
Would renaming the component `TestOption` and the props interface `TestOptionProps` make this more clear?
@gloriliale